### PR TITLE
Use object-fit for sponsor logos

### PR DESCRIPTION
**Pull request by haslinghuis**
- using object-fit for business logo's to ensures the image is fully visible within the aspect ratio box
- breakpoint on landing tab is 800px
- breakpoint on firmware flasher tab is 840px
- 4 logo's seems to fit on most mobile devices.

https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit

`Master`

image

`PR`

image

image

image

image

image

image

### CODE REVIEW

File: src/css/main.less [hunk 1]
```diff
@@ -1165,9 +1165,10 @@ dialog {
     min-height: 65px;
 }
 .img_sponsor {
-    height: 100%;
-    max-height: 50px;
+    object-fit: contain;
     margin: 5px 15px;
+    width: 150px;
+    height: 50px;
```
Comment (KarateBrot) [inline src/css/main.less:1171]:
```suggestion
    width: 150px;
    height: 45px;
```
The aspect ratio of the logos is 800/160 (=150/45)

**Review by KarateBrot — DISMISSED** [review]

File: src/css/main.less (hunk 1 above)
Comment (haslinghuis) [reply src/css/main.less]: Thanks. Fixed.

Comment (KarateBrot) [reply src/css/main.less]: I think now we also need to adjust line 1165 to
```js
min-height: 60px;
```

And limit the max-height in `.img_sponsor` to 45px

Comment (haslinghuis) [reply src/css/main.less]: Done.

**Review by SupaflyFPV — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Pull request merged**